src/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -142,4 +142,11 @@
     <protected>false</protected>
     <shortDescription>Abandon Request</shortDescription>
   </labels>
+  <labels>
+    <fullName>cookbook_Navigate</fullName>
```

Comment:
I'd suggest to have localized strings for `Previous` and `Next`. When `type` is set to `previous`, then the button text should default to `cookbook_Previous`. Same thing for `type='next'` where it should default to `cookbook_Next`.